[2024-11-13T01:38:47.311+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-13T01:38:47.368+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: PopulationDataToSnowflake.load_population_data manual__2024-11-13T01:38:38.078161+00:00 [queued]>
[2024-11-13T01:38:47.388+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: PopulationDataToSnowflake.load_population_data manual__2024-11-13T01:38:38.078161+00:00 [queued]>
[2024-11-13T01:38:47.398+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-11-13T01:38:47.422+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): load_population_data> on 2024-11-13 01:38:38.078161+00:00
[2024-11-13T01:38:47.467+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=150) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-13T01:38:47.476+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'PopulationDataToSnowflake', 'load_population_data', 'manual__2024-11-13T01:38:38.078161+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/PopulationDataToSnowflake.py', '--cfg-path', '/tmp/tmpi2iwfyj3']
[2024-11-13T01:38:47.478+0000] {standard_task_runner.py:63} INFO - Started process 152 to run task
[2024-11-13T01:38:47.480+0000] {standard_task_runner.py:91} INFO - Job 6: Subtask load_population_data
[2024-11-13T01:38:47.570+0000] {task_command.py:426} INFO - Running <TaskInstance: PopulationDataToSnowflake.load_population_data manual__2024-11-13T01:38:38.078161+00:00 [running]> on host 712e3f8fee64
[2024-11-13T01:38:47.690+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='PopulationDataToSnowflake' AIRFLOW_CTX_TASK_ID='load_population_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-13T01:38:38.078161+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-13T01:38:38.078161+00:00'
[2024-11-13T01:38:47.699+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-13T01:38:47.720+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn' for task execution.
[2024-11-13T01:38:47.723+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.10.25-linuxkit-aarch64-with-glibc2.36
[2024-11-13T01:38:47.731+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-13T01:38:49.841+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-11-13T01:38:50.102+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-11-13T01:39:40.636+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-11-13T01:39:40.643+0000] {logging_mixin.py:188} INFO - Successfully loaded 100 rows into dev.raw_data.population_data
[2024-11-13T01:39:40.645+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-11-13T01:39:40.646+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-13T01:39:40.666+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=PopulationDataToSnowflake, task_id=load_population_data, run_id=manual__2024-11-13T01:38:38.078161+00:00, execution_date=20241113T013838, start_date=20241113T013847, end_date=20241113T013940
[2024-11-13T01:39:40.780+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-11-13T01:39:40.815+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-13T01:39:40.819+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
